src/Compilers/Test/Core/CompilationVerifier.cs

```diff
@@ -456,27 +459,29 @@
             return _compilation.Assembly.Identity.GetDisplayName();
         }
 
+        /// <summary>
+        /// Obsolete. Use <see cref="VerifyMethodBody(string, string, bool, string, int)"/> instead.
+        /// </summary>
         public CompilationVerifier VerifyIL(
             string qualifiedMethodName,
-            XCData expectedIL,
+            string expectedIL,
             bool realIL = false,
             string sequencePoints = null,
             [CallerFilePath] string callerPath = null,
-            [CallerLineNumber] int callerLine = 0)
+            [CallerLineNumber] int callerLine = 0,
+            string source = null)
         {
-            return VerifyILImpl(qualifiedMethodName, expectedIL.Value, realIL, sequencePoints, callerPath, callerLine, escapeQuotes: false);
+            return VerifyILImpl(qualifiedMethodName, expectedIL, realIL, sequencePoints != null, source != null, callerPath, callerLine, escapeQuotes: false);
```

Comment:
>sequencePoints != null, source != null [](http://example.com/codeflow?start=73&length=38)

nit: consider naming those two arguments (`sequencePoints:` and `sequencePointsSource:`)